### PR TITLE
BDOG-733 update to play 2.6

### PR DESCRIPTION
**Pull request by christopherjturner**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  val bootstrapPlayVersion = "1.6.0"
+
+  val compile = Seq(
+    ws,
+    "uk.gov.hmrc"               %% "bootstrap-play-26" % bootstrapPlayVersion,
+    "org.apache.httpcomponents" %  "httpclient"        % "4.5.11"
+  )
+
+  def test(scope: String = "test,it") = Seq(
+    "uk.gov.hmrc"            %% "bootstrap-play-26"           % bootstrapPlayVersion % scope classifier "tests",
```
Comment (nigelhp) [inline project/AppDependencies.scala]: bootstrap-play-26 is a compile time dependency, so shouldn't need to be repeated here

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=0.13.17
+sbt.version=1.3.4
```
Comment (nigelhp) [inline project/build.properties]: any reason why not 1.3.8?

**Review by nigelhp — DISMISSED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (christopherjturner) [reply project/AppDependencies.scala]: good spot, will remove

File: project/build.properties (hunk 2 above)
Comment (christopherjturner) [reply project/build.properties]: will bump version

**Review by nigelhp — APPROVED** [review]

**Pull request merged**